configure

```diff
@@ -804,6 +805,25 @@ if enabled_or_auto libsystemd_daemon; then
   fi
 fi
 
+#
+# year2038
```

Comment:
I think from an ordering pov, it would be nicer to have this before all other time functions, no?